File: app/src/main/java/com/example/dpstuffproviderstore/models/OrderData.kt
package com.example.dpstuffproviderstore.models

import com.google.gson.annotations.SerializedName

data class OrderData (
        @SerializedName("id") var id : Int?,
        @SerializedName("address") var address : String?,
        @SerializedName("lastName") var lastName : String?,
        @SerializedName("firstName") var firstName : String?,
        @SerializedName("phone") var phone : String?,
        @SerializedName("frontDoor") var frontDoor : Int?,
        @SerializedName("apartNum") var apartNum : Int?,
        @SerializedName("floorNum") var floorNum : Int?,
        @SerializedName("intercom") var intercom : String?,
        @SerializedName("deliveryDate") var deliveryDate : String?,
        @SerializedName("timeFrom") var timeFrom : String?,
        @SerializedName("timeTo") var timeTo : String?,
        @SerializedName("commentary") var commentary : String?,
        @SerializedName("idCourier") var idCourier : String?,
        @SerializedName("summ") var summ : Double?,
        @SerializedName("priority") var priority : Int?,
        @SerializedName("status") var status : String?,
        @SerializedName("listProducts") var listProducts : List<ProductData>?
)

File: DPSP Api v2/Models/Data/OrderData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DPSPApiV2.Models.Data
{
    public class OrderData
    {
        public int OrderId { get; set; }
        public DateTime OrderDateTime { get; set; }
        public string DeliveryDate { get; set; }
        public string DeliveryTimeFrom { get; set; }
        public string DeliveryTimeTo { get; set; }
        public string Commentary { get; set; }
        public float Summ { get; set; }
        public string CodeToFinish { get; set; }
        public string Status { get; set; }
        public int ClientId { get; set; }
        public string ClientName { get; set; }
        public int CourierId { get; set; }
        public string CourierName { get; set; }
        public int? AddressId { get; set; }
        public string Address { get; set; }
        public int FrontDoor { get; set; }
        public int ApartmentNum { get; set; }
        public int FloorNum { get; set; }
        public string Intercom { get; set; }
        public List<OrderComposData> OrderProducts { get; set; }
    }
}
